packages/dds/register-collection/src/interfaces.ts

```diff
@@ -17,20 +12,9 @@ import { ISharedObject, ISharedObjectEvents } from "@fluidframework/shared-objec
  * Extends the base IChannelFactory to return a more definite type of IConsensusRegisterCollection
  * Use for the runtime to create and load distributed data structure by type name of each channel.
  * @alpha
+ * @deprecated Use IChannelFactory<IConsensusRegisterCollection>.
```

Comment:
Nit: would recommend either linking to each of the types, or at least wrapping the type block in back-ticks.